docs/about.rst

```diff
@@ -35,27 +35,9 @@ science (radiative transfer simulations for instance).  Hence
 Contributors
 ============
 
-Alphabetical by last name:
-
-* Stephen Bailey
-* Kyle Barbary
-* Tom Barclay
-* Rahul Biswas
-* Kyle Boone
-* Matt Craig
-* Ulrich Feindt
-* Brian Friesen
-* Danny Goldstein
-* Saurabh Jha
-* David Jones
-* Pierre-Francois Leget
-* Florian Mondon
-* Daniel Perrefort
-* Justin Pierel
-* Benjamin Rose
-* Steve Rodney
-* Clare Saunders
-* Caroline Sofiatti
-* Rollin C. Thomas
-* Maria Vincenzi
-* Michael Wood-Vasey
+SNCosmo was originally written by Kyle Barbary in 2012. Since then, over 20 people have
+contributed to its development. A full list of contributors can be found in the
+`CITATION.cff <https://github.com/sncosmo/sncosmo/blob/master/CITATION.cff>`_ file.
+Parsed versions of the author list, including BibTeX entries that can be used to cite
+SNCosmo, are available on `GitHub <https://github.com/sncosmo/sncosmo>`_ and `Zenodo
+<https://doi.org/10.5281/zenodo.592747>`_.
```

Comment:
Does Zenodo parse the cff file? In other words, will the authors listing on the Zenodo page update as the cff file updates?

Reply:
Yes, Zenodo parses CFF files. See https://citation-file-format.github.io/

Reply:
My bad, thanks.

I have read that page several times, but did not pay attention to the Zenodo section. I assumed it just said "hey, they use this format too" and not "They will use the specific file from GitHub." Nice to know, and one less thing to update once we adopt this format.